src/modules/instance/index.js

```diff
@@ -32,8 +28,7 @@ const rootReducer = (state, action) => {
     return {
       ...state,
       errorMessage: ERROR_MESSAGE_INITIAL_STATE,
-      instance: INSTANCE_INITIAL_STATE,
-      dataService: DATA_SERVICE_INITIAL_STATE
+      instance: global.hadronApp.instance // TODO: reset??
```

Comment:
@durran Does it make sense to keep the instance as the initial InstanceModel? Otherwise would be concerned that the state would be a non-ampersand model and could cause an error.